assemblies/DataSources/OpenConfig/Skyline.DataMiner.DataSources.OpenConfig.Gnmi.xml

```diff
@@ -449,44 +401,57 @@
+            Processes the gNMI <see cref="T:Gnmi.SubscribeResponse"/> and adapts the <see cref="T:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.DataMapper.IDataEntity"/> values accordingly.
             </summary>
-            <param name="notification">gNMI notification to be processed.</param>
+            <param name="subscribeResponse">gNMI subscribe response to be processed.</param>
             <param name="origin">Indication of where the data originated from.</param>
-            <param name="isSyncResponse">Boolean indicating that the initial transmission of updates has concluded.</param>
             <param name="subscriptionName">Name of the subscription under which the notification entered.</param>
         </member>
         <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.DataMapper.IDataMapper.ReadExistingValues">
             <summary>
             Reads out the existing values and stores them internally.
             </summary>
         </member>
-        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.DataMapper.IDataMapper.RegisterSubscriptionPaths(System.String,System.Collections.Generic.IEnumerable{Gnmi.Path})">
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.DataMapper.IDataMapper.RegisterSubscriptionPaths(System.String,System.Collections.Generic.IEnumerable{Gnmi.Path},System.Action{System.Collections.Generic.IEnumerable{Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Models.GnmiResponseValue}},Gnmi.SubscriptionMode)">
             <summary>
             Registers the <see cref="T:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.DataMapper.IDataEntity"/> objects that are linked with a subscription name.
             </summary>
             <param name="subscriptionName">Name of the subscription.</param>
             <param name="paths">List of paths that are subscribed.</param>
+            <param name="action">The action that needs to be executed when a response comes in.</param>
+            <param name="subscriptionMode">The mode of the subscription.</param>
         </member>
-        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.DataMapper.IDataMapper.SetLogger(Skyline.DataMiner.DataSources.OpenConfig.Gnmi.Utils.ILogger)">
+        <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.DataMapper.IDataMapper.SetLogger(Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Api.ILogger)">
             <summary>
             Assigns a logger interface.
             </summary>
             <param name="logger">Logger interface</param>
         </member>
         <member name="M:Skyline.DataMiner.DataSources.OpenConfig.Gnmi.DataMapper.IDataMapper.SuppressCommunication(System.Boolean)">
             <summary>
-            Suppresses the communication, no more values will be filled in or logging will be written out.
+            Suppresses the communication. No more values will be filled in or logging will be written out.
```

Comment:
Similar question as before: Do you mean no more values will be filled in and no logging will be written out?